### ISSUE TITLE
Allow installation through homebrew

### ISSUE THREAD

**Issue by daften** [kriskbx/gitlab-time-tracker#69]
Started this: https://github.com/Homebrew/homebrew-core/pull/32154#pullrequestreview-155949702

However it fails, issue in commander opened: https://github.com/tj/commander.js/issues/866

**Comment by kriskbx** [comment]
Very cool. Thank you! I'm working on a continuos integration pipeline to create binaries and push them to GitHub releases right now. I guess this would help, wouldn't it?

**Comment by daften** [comment]
yes, homebrew formula's and casks still need manual updates, but that's fairly easy to do. The PR for the gtt-taskbar app wasn't accepted though (https://github.com/Homebrew/homebrew-cask/pull/52100), so i created my own tap atm: https://github.com/daften/homebrew-custom.

**Comment by kriskbx** [comment]
Starting with version 1.7.30 precompiled binaries for Mac/Win/Linux (x64) are available via GitHub releases. 😃

**Comment by daften** [comment]
I've updated the PR for this. It would be nice if the binaries were executable by default, if that's possible.

**Comment by daften** [comment]
I applaud your effort, however, this is what needs to be done: "Please revert the last commit, Homebrew formulae are required to build from source." (https://github.com/Homebrew/homebrew-core/pull/32154#issuecomment-426980008)

**Comment by kriskbx** [comment]
I will keep the binaries as a convenient way for users to download and install gtt when experiencing problems with other installation methods. (Will update Readme soon)
Anyway: Thank you so much for your effort to get the Homebrew formulae up and working. If there's anything I can do don't hesitate to ask, but I guess we have to wait for the commander and/or yarn team to fix this.

**Comment by daften** [comment]
Yeah, makes sense. No worries, I'll get everything ready so when command fixes that issue (somebody proposed to make a PR 3 days ago), we can get that in. Thanks for this tool, saves me a lot of time ;)

**Comment by kriskbx** [comment]
Me too, that's why I built it 😄 You're welcome!

**Comment by kriskbx** [comment]
I made some changes today by forking the commander library and fixing it. All issues with `gtt-config(1) does not exist, try --help` should be resolved by now.

**Comment by daften** [comment]
I won't submit it to the official repository again, but I created a formula in a custom tap: https://github.com/daften/homebrew-custom/blob/master/Formula/gitlab-time-tracker.rb

From that it should be fairly easy to create a formula for the official repo if somebody wants to do this. It works now, so I'll close this.

Thanks @kriskbx
